modules/install-sno-ibm-z-lpar.adoc

```diff
@@ -0,0 +1,226 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-sno-on-ibm-z-lpar_{context}"]
+= Installing {sno} in an LPAR on {ibm-z-title} and {ibm-linuxone-title}
+
+.Prerequisites
+
+* If you are deploying a single-node cluster there are zero compute nodes; the Ingress Controller pods run on the control plane nodes. In single-node cluster deployments, you must configure your application ingress load balancer to route HTTP and HTTPS traffic to the control plane nodes. See the _Load balancing requirements for user-provisioned infrastructure_ section for more information.
+
+.Procedure
+
+. Set the {product-title} version by running the following command:
++
+[source,terminal]
+----
+$ OCP_VERSION=<ocp_version> <1>
+----
++
+<1> Replace `<ocp_version>` with the current version. For example, `latest-{product-version}`.
+
+. Set the host architecture by running the following command:
++
+[source,terminal]
+----
+$ ARCH=<architecture> <1>
+----
+<1> Replace `<architecture>` with the target host architecture `s390x`.
+
+. Download the {product-title} client (`oc`) and make it available for use by entering the following commands:
```

Comment:
I see that this is following the model of the existing content. It would be good to update all three to use a single command per step, but that is out of scope here. Suggest adding it as a backlog docs enhancement.

Reply:
Will add this and the next comment to the post GA backlog as this applies to x86 procedures too.